etc/config/c.amazon.properties

```diff
@@ -3161,6 +3161,15 @@ libs.libuv.versions.1381.version=1.38.1
 libs.libuv.versions.1381.path=/opt/compiler-explorer/libs/libuv/v1.38.1/include
 libs.libuv.versions.1381.libpath=/opt/compiler-explorer/libs/libuv/v1.38.1/x86_64/lib:/opt/compiler-explorer/libs/libuv/v1.38.1/x86/lib
 
+libs.liblzma.name=liblzma
+libs.liblzma.description=XZ-format compression library
+libs.liblzma.liblink=lzma
+libs.liblzma.url=https://tukaani.org/xz/
+libs.liblzma.versions=562
+libs.liblzma.versions.562.version=5.6.2
+libs.liblzma.versions.562.path=/opt/compiler-explorer/libs/liblzma/v5.6.2/include
```

Comment:
I'm going to merge and patch up based on how I think this needs to go

Reply:
Thank you Matt for the handholding - I didn't expect it to be so much hassle!